docs/tasks.md

```diff
@@ -26,6 +26,14 @@ Discover PDRs in FTP and HTTP endpoints
 
 ---
 
+### [@cumulus/files-to-granules](https://github.com/nasa/cumulus/tree/master/tasks/post-to-cmr)
```

Comment:
Is this supposed to point at `post-to-cmr`?

Reply:
Huh, was this generated via build-tasks-doc?

Reply:
It was a change after I ran `docs-serve`, which presumably ran `build-tasks-doc` since this was the first time I ran it since my system reimage. Not sure why it would point to the wrong task.

Reply:
a646bcf422b4bd66a7db6a80e32cd43e21eb089f